openprescribing/dmd/migrations/0004_auto_20171129_1341_squashed_0008_auto_20171129_1354.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
```

Comment:
A small thing for neatness: unless this has already been applied, you could delete this and the next migration and re-run `makemigrations`.  You'd need to roll back any migrations locally.